src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/self-closing-comp */
+import React from 'react';
+import './Tabs.scss';
+
+interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
+
+type Selection = (params: Tab) => void;
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: Selection,
+};
+
+const Tabs: React.FC<Props> = ({ tabs, selectedTabId, onTabSelected }) => {
+  return (
+    <>
+      <div
+        className="ButtonsContainer"
+      >
+        {
+          tabs.map(tab => (
+            <>
```

Comment:
key надо в корневой элемент прописывать, в данном случае это Fragment, и тебе надо записывать так:
```suggestion
            <React.Fragment key={tab.id}>
```